### DISCUSSION TITLE
Opening specific files on gui applications

### DISCUSSION THREAD

**Discussion by jabarragann** [jarun/nnn#1659] [General]
Hi there,

I have configured NNN to open all my files in the terminal. The only exception to this rule is that I want to be able to open pdf files with my default desktop GUI application. How could I configure NNN to do this?

To be more precise:
I open `nnn` with `-ce` option to have most of the file extensions open in the terminal. Now currently, if I press `l`  on a pdf file, nnn will open it in my terminal editor. How can I overwrite this behavior only for the specific .pdf files?

Now, If I remove the `-c` option, everything is going to be opened in GUI applications.

**Comment by amalgame21** [comment]
`NNN_OPENER`  using `nuke`?

https://github.com/jarun/nnn/blob/fb6e7403cf3c1482c8a5ad12e7cfed5e978127cd/plugins/nuke#L67-L69

https://github.com/jarun/nnn/blob/fb6e7403cf3c1482c8a5ad12e7cfed5e978127cd/plugins/nuke#L90-L110

**Comment by jarun** [comment]
Yes, please use a custom opener.
